<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZEMLEditorWindow</class>
 <widget class="QMainWindow" name="ZEMLEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZEML Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTreeWidget" name="trwElementTree">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="ZEMLEditorElementWidget" name="wgtElementEditor" native="true"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actNew"/>
    <addaction name="separator"/>
    <addaction name="actOpen"/>
    <addaction name="separator"/>
    <addaction name="actSave"/>
    <addaction name="actSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actClose"/>
    <addaction name="separator"/>
    <addaction name="actQuit"/>
   </widget>
   <widget class="QMenu" name="menuElements">
    <property name="title">
     <string>Elements</string>
    </property>
    <addaction name="actAddNode"/>
    <addaction name="actAddProperty"/>
    <addaction name="actAddData"/>
    <addaction name="separator"/>
    <addaction name="actDelete"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actUndo"/>
    <addaction name="actRedo"/>
    <addaction name="separator"/>
    <addaction name="actCut"/>
    <addaction name="actCopy"/>
    <addaction name="actPaste"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actUserGuide"/>
    <addaction name="separator"/>
    <addaction name="actWebsite"/>
    <addaction name="actBugReport"/>
    <addaction name="separator"/>
    <addaction name="actAbout"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actImport"/>
    <addaction name="separator"/>
    <addaction name="actSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuElements"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actSaveAs">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actAddNode">
   <property name="text">
    <string>Add Node</string>
   </property>
  </action>
  <action name="actAddProperty">
   <property name="text">
    <string>Add Property</string>
   </property>
  </action>
  <action name="actAddData">
   <property name="text">
    <string>Add Data</string>
   </property>
  </action>
  <action name="actDelete">
   <property name="text">
    <string>Delete Element(s)</string>
   </property>
  </action>
  <action name="actUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actUserGuide">
   <property name="text">
    <string>User Guide</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actBugReport">
   <property name="text">
    <string>Bug Report</string>
   </property>
  </action>
  <action name="actWebsite">
   <property name="text">
    <string>Website</string>
   </property>
  </action>
  <action name="actSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actImport">
   <property name="text">
    <string>Combine Files</string>
   </property>
  </action>
  <action name="actExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZEMLEditorElementWidget</class>
   <extends>QWidget</extends>
   <header>ZEMLEditorElementWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
